packages/Babylonian-Core.package/BPAssertion.class/instance/label.st

```diff
@@ -0,0 +1,4 @@
+accessing
+label
+
+	^ label
```

Comment:
`hasLabel`, `label` and `label:` are duplicated to `BPProbe`. Maybe we should move it up to `BPAnnotation?`

Reply:
Would have moved it there if >2 annotation types use a label.

Reply:
@JoeAtHPI: I had the same idea at first, but I aggree with Lina, that there are only two subclasses using it right now. If we find a third one, we can refactor it.